src/metabase/api/query_description.clj

```diff
@@ -14,13 +14,21 @@
 
 (defn- get-aggregation-description
   [metadata query]
-  (let [field-name (fn [match] (:display_name (Field (mbql.u/field-clause->id-or-literal match))))]
+  (let [field-name (fn [match] (map
```

Comment:
Why is this a local function? It doesn't seem to close over anything.

Also, since we're only interested in ids, it might be clearer to capture that explicitly:
```
(mbql.u/match match
  [:field-id id] (-> id Field :display_name))
```